src/client/configClient.cfg

```diff
@@ -29,6 +29,7 @@ systems = (
             title = "RType",
             texturesPath = "./config/textures.cfg",
             shadersPath = "./config/shaders.cfg",
+            defaultShader = "protanopia",
```

Comment:
default value should not be a filter I think

Reply:
Indeed, it has been set back to default